core/src/test/scala/unit/kafka/admin/ListConsumerGroupTest.scala

```diff
@@ -55,7 +55,11 @@ class ListConsumerGroupTest extends KafkaServerTestHarness {
   def testListGroupWithNoExistingGroup() {
     val opts = new ConsumerGroupCommandOptions(Array("--zookeeper", zkConnect))
     val consumerGroupCommand = new ZkConsumerGroupService(opts)
-    assert(consumerGroupCommand.listGroups().isEmpty)
+    try {
+      assert(consumerGroupCommand.listGroups().isEmpty)
```

Comment:
I didn't change this in the end as it's using a ScalaTest assert instead of the built-in Scala assert. This is a bit confusing and it may be worth changing in a separate PR.